Airbnb.Domain/Entities/Owner.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Airbnb.Domain.Entities
+{
+    public class Owner:BaseEntity<int>
+    {
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string PhoneNumber { get; set; }
```

Comment:
what if we make a relationship between the user and property directly instead of creating another data ? is it a good solution?